test/parallel/test-tcp-wrap-connect.js

```diff
@@ -18,20 +18,20 @@ function makeConnection() {
   assert.strictEqual(err, 0);
 
   req.oncomplete = function(status, client_, req_, readable, writable) {
-    assert.strictEqual(0, status);
-    assert.strictEqual(client, client_);
-    assert.strictEqual(req, req_);
-    assert.strictEqual(true, readable);
-    assert.strictEqual(true, writable);
+    assert.strictEqual(status, 0);
+    assert.strictEqual(client_, client);
+    assert.strictEqual(req_, req);
+    assert.strictEqual(readable, true);
+    assert.strictEqual(writable, true);
 
     const shutdownReq = new ShutdownWrap();
     const err = client.shutdown(shutdownReq);
     assert.strictEqual(err, 0);
 
     shutdownReq.oncomplete = function(status, client_, error) {
-      assert.strictEqual(0, status);
-      assert.strictEqual(client, client_);
-      assert.strictEqual(error, undefined);
```

Comment:
Should this order be preserved?

Reply:
Hi, @trivikr , would you explain further you comment? I believe that the order here is already changed, unless you meant something else.

Reply:
My bad, the comment was for line 34.
The strictEqual assertions should follow the order `actual-value` -> `expected-value`, where `error` is actual value, while `undefined` is expected value.

Reply:
Fixed while landing.